eva-accession-import/README.md

```diff
@@ -0,0 +1,25 @@
+# EVA accession import
+
+This module can be used to import accessions (SS and RS) from a dbsnp mirror into the accessioning service at EVA. In https://github.com/EBIvariation/eva-accession/wiki there's some documentation about the transformations done.
```

Comment:
`dbSNP mirror`

`Documentation about the transformations applied can be found in https://github.com/EBIvariation/eva-accession/wiki`